src/PscIde/Command.purs

```diff
@@ -283,8 +283,8 @@ data ImportType = Implicit | Explicit (Array String) | Hiding (Array String)
 data ImportResult = SuccessFile Message | SuccessText (Array String) | MultipleResults (Array TypeInfo)
 
 unwrapResponse :: forall a b. DecodeJson a => DecodeJson b => String -> Either String (Either a b)
-unwrapResponse s = do
-  json <- jsonParser s
+unwrapResponse s = lmap printJsonDecodeError do
```

Comment:
I opted to recover the string here as this is used in the later `makeAff` calls. To minimize changes I thought this was the appropriate place to return from `JsonDecodeError` to the previous `String` type.